content/docs/operations/install.md

```diff
@@ -89,15 +89,15 @@ Install the open source {{< reuse "docs/snippets/product-name.md" >}} project in
 2. Optional: Pull and inspect the {{< reuse "docs/snippets/product-name.md" >}} Helm chart values before installation. You might want to update the Helm chart values files to customize the installation. For example, you might change the namespace, update the resource limits and requests, or enable extensions such as for AI.
    
    ```sh
-   helm pull oci://ghcr.io/kgateway-dev/charts/kgateway --version v{{< reuse "docs/versions/n-patch.md" >}}
+   helm pull oci://cr.kgateway.dev/charts/kgateway --version v{{< reuse "docs/versions/n-patch.md" >}}
```

Comment:
should there still be a `kgateway-dev`?
```suggestion
   helm pull oci://cr.kgateway.dev/kgateway-dev/charts/kgateway --version v{{< reuse "docs/versions/n-patch.md" >}}
```

Reply:
confirmed, they should all be `oci://cr.kgateway.dev/kgateway-dev/charts/kgateway`